src/nnet3/nnet-am-decodable-simple.h

```diff
@@ -75,6 +77,13 @@ struct NnetSimpleComputationOptions {
                    "input frames");
     opts->Register("debug-computation", &debug_computation, "If true, turn on "
                    "debug for the actual computation (very verbose!)");
+    opts->Register("ivector-interval", &ivector_interval, "Specify the interval "
```

Comment:
If non-zero it is the interval between two ivectors used during decoding. It is recommended to use the same value as the one used for training.